src/openlifu/db/database.py

```diff
@@ -377,22 +392,11 @@ def load_subject(self, subject_id, options=None):
         self.logger.info(f"Loaded subject {subject_id}")
         return subject
 
-    def load_volume(self, subject, volume_id):
-        raise NotImplementedError("Volume is not yet implemented")
-        volume_filename = self.get_volume_filename(subject.id, volume_id)
+    def load_volume(self, subject_id, volume_id):
```

Comment:
This should not be called `load_volume` because that sounds like it is  loading a volume file, but it is not. It is loading just metadata and a filepath. Maybe `load_volume_info` or `get_volume_info`